terraso_backend/config/settings.py

```diff
@@ -365,3 +366,19 @@
     "/healthz/",
     "/web-client/sitemap.xml",
 ]
+
+
+class JWTProvider(TypedDict):
+    """Type hint to indicate correct config for JWT_EXCHANGE_PROVIDERS"""
+
+    url: str
+    client_id: str
+    name: str
+
+
+JWT_EXCHANGE_PROVIDERS: dict[str, JWTProvider] = {
+    "google": dict(
+        url="https://www.googleapis.com/oauth2/v3/certs",
+        client_id=config("GOOGLE_MOBILE_CLIENT_ID", default=""),
```

Comment:
question: Do we need a default? Shouldn't the config file gave a fallback value already?

Reply:
Yes, but the config values aren't read in unit tests, which causes an exception to be thrown when the env variable doesn't have a value. We could make a separate issue to create an auto-loaded pytest fixture that loaded the env file if we think it would be useful.